Source/cmGlobalFastbuildGenerator.h

```diff
@@ -55,6 +55,12 @@ class cmGlobalFastbuildGenerator
 	virtual void ComputeTargetObjectDirectory(cmGeneratorTarget*) const;
 	virtual const char* GetCMakeCFGIntDir() const;
 
+	virtual void GetTargetSets(TargetDependSet& projectTargets,
+							   TargetDependSet& originalTargets,
+							   cmLocalGenerator* root, GeneratorVector const&);
+
+	const std::vector<std::string> & GetConfigurations() const;
```

Comment:
Is this needed?
Everything in the C++ should have 100% access to the members of the global generator...